.github/workflows/coveralls.yml

```diff
@@ -64,11 +64,13 @@ jobs:
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4} TESTS='-test_external_krb5'
     - name: generate coverage info
-      run: lcov -d . -c -o ./lcov.info
+      run: |
+        lcov -d . -c -o ./lcov.info
+        lcov --remove lcov.info ${PWD}/test/* ${PWD}/test/helpers/* ${PWD}/test/testutil/* -o lcov-filtered.info
```

Comment:
Can this be done in one pass with the `--exclude` option to lcov?

Reply:
> Can this be done in one pass with the `--exclude` option to lcov?

Let me try that.

Reply:
Done.